docs/gtk/install.md

```diff
@@ -119,108 +76,93 @@ Now create your super-application myfoo.cpp and compile anywhere with
 
     g++ myfoo.cpp `wx-config --libs --cxxflags` -o myfoo
 
-GUI libraries                          {#gtk_libs_gui}
----------------
-
-wxWidgets/GTK requires the GTK library to be installed on your system.
-
-You can get the newest version of GTK from the GTK homepage at:
 
-  http://www.gtk.org
-
-Additional libraries                   {#gtk_libs_misc}
-----------------------
-
-wxWidgets/Gtk requires a thread library and X libraries known to work with
-threads.  This is the case on all commercial Unix-Variants and all
-Linux-Versions that are based on glibc 2 except RedHat 5.0 which is broken in
-many aspects. As of writing this, virtually all Linux distributions have
-correct glibc 2 support.
-
-You can disable thread support by running
-
-    ./configure --disable-threads
-    make
-    su <type root password>
-    make install
-    ldconfig
-    exit
 
 Building wxGTK on Cygwin               {#gtk_cygwin}
 --------------------------
 
 The normal build instructions should work fine on Cygwin. The one difference
 with Cygwin is that when using the "--enable-shared" configure option (which
-is the default) the API is exported explicitly using __declspec(dllexport)
+is the default) the API is exported explicitly using `__declspec(dllexport)`
 rather than all global symbols being available.
 
 This shouldn't make a difference using the library and should be a little
 more efficient. However if an export attribute has been missed somewhere you
 will see linking errors. If this happens then you can work around the
-problem by setting LDFLAGS=-Wl,--export-all-symbols. Please also let us know
+problem by setting `LDFLAGS=-Wl,--export-all-symbols`. Please also let us know
 about it on the wx-dev mailing list.
 
-Create your configuration              {#gtk_config}
----------------------------
 
-Usage:
+Configure options                        {#gtk_options}
```

Comment:
The configure option for disabling the EGL backend and using the X11 backend instead for the OpenGL canvas should be added to this list.

Reply:
Thanks, good idea, added now.